omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
     $ javac -version
     javac 1.8.0_51
 
-Requirements
-------------
 
-.. _`Homebrew wiki`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md
+OS X Basics
+-----------
 
-All the requirements for OMERO will be installed under :file:`/usr/local`. See also: Installation instructions on the `Homebrew wiki`_.
+In order to develop on OMERO, we recommend you ensure you have your Mac setup for
+development. The first step to achieving this is to create a :file:`.bash_profile` file in the
+root directory of your user folder.
 
-Install Homebrew and make sure :file:`/usr/local/bin` is prepended to your
-:envvar:`PATH`::
+To create a :file:`.bash_profile` from terminal, if one does not already exist::
 
-    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-    $ export PATH=/usr/local/bin:$PATH
+    $ touch ~/.bash_profile
 
-Update Homebrew and run 'brew doctor' to fix potential issues beforehand::
+To open your :file:`.bash_profile` in a text editor, such as the built-in TextEdit app, use::
 
-    $ brew update
-    $ brew doctor
+    $ open -a TextEdit.app ~/.bash_profile
 
-Install git if not already present::
+.. note::
+   If you want to see changes to your :file:`.bash_profile` take effect without restarting
```

Comment:
``...to see changes ... take effect..`` -> maybe ``...want changes to your :file:`.bash_profile` take effect...`` (the original sentence seems unnecessary complicated with the ``see``)